src/Stratis.Bitcoin.IntegrationTests/Wallet/SendingRecentlyStakedCoinsBeforeMaturity.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Xunit;
+
+namespace Stratis.Bitcoin.IntegrationTests.Wallet
+{
+    public partial class SendingRecentlyStakedCoinsBeforeMaturity
+    {
+        [Fact]
+        public void Sending_Staked_Coins_Before_Maturity_Is_Not_Reflected_In_The_Wallets_Transaction_History()
+        {
+            Given(two_nodes_which_includes_a_proof_of_stake_node_with_a_million_coins);
+            When(a_wallet_sends_staked_coins_before_maturity);
+            Then(the_wallet_history_shows_the_transaction_as_sent);
```

Comment:
I can't see how this ties in with the title of the test.  Title says Not reflected in history, but the Then says it is.

Reply:
Changed to this "the_wallet_history_shows_the_transaction_as_sent_but_the_transaction_was_not_received".  Is this better ?